VideoSyncClass/VideoSync.py

```diff
@@ -0,0 +1,283 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from subprocess import run, call
+from pathlib import Path
+import yaml
+import os
+from scipy.io import wavfile
+from scipy.io.wavfile import read as wavread
+import mne
+
+class VideoSync:
+    def __init__(self, input_folder=None, output_folder="/output"):
+        """
+        Summary: VideoSync represents an object performing synchronization between multiple streams of data.
+
+        Attributes:
+            input_folder: Directory where data is supplied from.
+            output_folder: Directory where analysis files will be published to. Defaults to "input_folder/output"
+
+            self.video_paths: List of RELATIVE paths to video files.
+            self.audio_paths: List of RELATIVE paths to audio files.
+            self.meg_paths: List of RELATIVE paths to meg files.
+        """
+        # User defined paths for input/output directories
+        self.__input_folder = input_folder
```

Comment:
starting attributes/variable names with double-underscore is unusual. A leading single underscore for "private" functions/classes/attributes is common.